core/src/main/java/de/danoeh/antennapod/core/storage/DBUpgrader.java

```diff
@@ -288,6 +288,10 @@ static void upgrade(final SQLiteDatabase db, final int oldVersion, final int new
 
             db.execSQL("DROP TABLE " + PodDBAdapter.TABLE_NAME_FEED_IMAGES);
         }
+        if (oldVersion < 1070296) {
```

Comment:
Please change to `1070400`. I promise to look into this PR in more detail before the 1.7.4 release ;)